app/livechat/server/api/v1/message.js

```diff
@@ -34,6 +34,10 @@ API.v1.addRoute('livechat/message', {
 				throw new Meteor.Error('invalid-room');
 			}
 
+			if (room && !room.open) {
```

Comment:
You can just check:

```
if (!room.open) {
...
}
```

Because the `room` object is checked here:
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/api/v1/message.js#L33

Reply:
cool, let me update it

Reply:
Done